discovery-provider/compose/.env

```diff
@@ -19,4 +19,5 @@ audius_solana_signer_group_address=CkEr7jVopeNDSYGQYuJnHYE8b62hisw5qCGBzf9Pwxev
 audius_solana_signer_group_address=CkEr7jVopeNDSYGQYuJnHYE8b62hisw5qCGBzf9Pwxev
 audius_solana_signer_group_address=CkEr7jVopeNDSYGQYuJnHYE8b62hisw5qCGBzf9Pwxev
 audius_solana_signer_group_address=CkEr7jVopeNDSYGQYuJnHYE8b62hisw5qCGBzf9Pwxev
-audius_solana_signer_group_address=CkEr7jVopeNDSYGQYuJnHYE8b62hisw5qCGBzf9Pwxev
\ No newline at end of file
+audius_solana_signer_group_address=CkEr7jVopeNDSYGQYuJnHYE8b62hisw5qCGBzf9Pwxev
```

Comment:
This is added 5 times

Reply:
yeah...this was on prod so i kept it the same, will update